josemak25/baby-bliss
# Ft get started screens

#### What does this PR do?
Have an overview feature of the app displayed in get started screen.

#### Description of Task to be completed?
Have the get started screen show the following:
1. Slider to show a brief description of core features of the app.
2. A slider pagination to show current slide position.
3. A get started button that leads to the sign up screen
4. A login button that leads to login screen

#### How should this be manually tested?
After cloning the repo, 
``CD`` into it and ``RUN`` 

```javascript
git checkout ft-get-started-screen

git pull origin  ft-get-started-screen

yarn install

yarn test
```
Using an emulator: Launch the app, immediately after the splash screen you should see the get started screen.

#### Screenshots (if appropriate)
![Simulator Screen Shot - iPhone Xs - 2020-02-24 at 09 56 16](https://user-images.githubusercontent.com/12932317/75139025-e9eabd80-56eb-11ea-944f-93bec2b9e9d1.png)
